<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_note_widget</class>
 <widget class="QWidget" name="add_note_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="5">
      <widget class="QLineEdit" name="lineEdit_type">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>类别</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_add_note">
       <property name="text">
        <string>新增</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="pushButton_add_type">
       <property name="text">
        <string>新增</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_value">
       <property name="maximum">
        <double>9999.989999999999782</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_note">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_type"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>金额</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>描述</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>类别</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_day">
     <property name="title">
      <string>今日消费 0</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableWidget" name="tableWidget_day">
        <column>
         <property name="text">
          <string>类别</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>金额</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>描述</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_add_note</sender>
   <signal>clicked()</signal>
   <receiver>add_note_widget</receiver>
   <slot>slot_add_note_bt_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>-5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_add_type</sender>
   <signal>clicked()</signal>
   <receiver>add_note_widget</receiver>
   <slot>slot_add_type_bt_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>744</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>794</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_add_note_bt_clicked()</slot>
  <slot>slot_add_type_bt_clicked()</slot>
 </slots>
</ui>
